config/crd/bases/operators.operatorframework.io_operators.yaml

```diff
@@ -39,6 +39,16 @@ spec:
                 maxLength: 48
                 pattern: ^[a-z0-9]+(-[a-z0-9]+)*$
                 type: string
+              version:
+                description: "Version is an optional is semver constraint on the package
```

Comment:
Same grammar nit.

Reply:
fixed ^^ nice catch. I also modified the "Example" because the rendering in the description looked funky